src/lib/npm-client.ts

```diff
@@ -92,6 +92,10 @@ export class UncachedNpmInfoClient {
 			if (info.error === "Not found") { return undefined; }
 			throw new Error(`Error getting version at ${escapedPackageName}: ${info.error}`);
 		}
+		if (!info["dist-tags"] && !info.versions) {
```

Comment:
When does this happen? I thought we get "error" if the package doesn't exist. If the package does exist it should have "versions".

Reply:
Yeah I thought so too but when I was trying to generate all packages it was trying to get the package info for `emscripten` which had *no* `versions`. See http://registry.npmjs.org/emscripten